pr2_moveit_tutorials/planning/scripts/move_group_python_interface_tutorial.py

```diff
@@ -186,7 +186,9 @@ def move_group_python_interface_tutorial():
   ## for the end-effector to go through.
   waypoints = []
 
-  # start with the current pose
+  # start with the current pose, only for rviz simulation. Remove this step if working
```

Comment:
There is no "RViz simulation".
You are talking about the MoveIt's demo(.launch) mode, aren't you?